sources/packages/web/src/views/institution/locations/active-applications/ActiveApplicationReportAChange.vue

```diff
@@ -30,12 +35,50 @@ export default {
     },
   },
   setup(props: any) {
-    const initialData = ref({});
+    const router = useRouter();
+    const initialData = ref({} as ApplicationDetails);
+    const loadInitialData = async () => {
+      const applicationDetails: ApplicationDetailsForCOEDTO = await ConfirmationOfEnrollmentService.shared.getApplicationForCOE(
+        props.applicationId,
+        props.locationId,
+      );
+      initialData.value = {
+        applicationProgramName: applicationDetails.applicationProgramName,
```

Comment:
If all the names are the same, could the code not directly assign the _applicationDetails_ to the _initialData.value_?

Reply:
there are some extra variables that are not needed for the form, here I am reusing the COE API